planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -482,6 +482,141 @@ PathWithLaneId NormalLaneChange::getPrepareSegment(
   return prepare_segment;
 }
 
+LaneChangeTargetObjects NormalLaneChange::getTargetObjects(
+  const PredictedObjects & objects, const lanelet::ConstLanelets & current_lanes,
```

Comment:
@purewater0901 
I would recommend just remove parameter `objects` and just call `planner_data->dynamic object directly in this function.

Reply:
done with https://github.com/autowarefoundation/autoware.universe/pull/4273/commits/82c4c8c2787bd902aef7d3e28693971e1f6a4935